lib/location.ex

```diff
@@ -7,18 +9,23 @@ defmodule Location do
   defdelegate get_city(code), to: Location.City
   defdelegate get_city(city_name, country_code), to: Location.City
 
-  def load_all(timeout \\ 30_000) do
-    me = self()
-
+  def load_all() do
```

Comment:
I'm removing the timeout because there was still no way to recover from it and it was crashing the main app startup 🤷